{
    "directory name": "llvm-project/lldb/test/API/lang/cpp/exceptions",
    "score.sum": 0.000907563336236955,
    "sha.latest": "80fcecb13c388ff087a27a4b0e7ca3dd8c98eaa4",
    "time.latest": 1717931126.8536665,
    "formatted_time": "2024-06-09 16:35:26.853668",
    "filescores": {
        "exceptions.cpp": 0.0,
        "Makefile": 0.0,
        "TestCPPExceptionBreakpoints.py": 0.057440614682545625
    },
    "subdirscores": {}
}